tests/integration/components/guides-article-test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable ember/avoid-leaking-state-in-ember-objects */
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import Service from '@ember/service';
+
+const GH_LINK = 'https://github.com/ember-learn/guides-source/edit/master/guides';
+
+const ARTICLE = `
+\`\`\`javascript {data-filename="tests/acceptance/list-rentals-test.js" data-diff="+7,-8,+9,-10,+11,-12,+13"}
+import { module, test } from 'qunit';↵import { setupApplicationTest } from 'ember-qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+
+module('Acceptance | my acceptance test', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /list-rentals', async function(assert) {
+  test('visiting /', async function(assert) {
+    await visit('/list-rentals');
+    await visit('/');
+    assert.equal(currentURL(), '/list-rentals');
+    assert.equal(currentURL(), '/');
+  });
+});
+\`\`\`
+`;
+
+module('Integration | Component | guides-article', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it shows basic article information', async function(assert) {
+    this.set('version', 'v3.2.0');
+    this.set('currentVersion', 'v3.2.0');
+    this.set('model', {
+      content: 'Oh hi',
+      id: 123
+    });
+
+    let pageServiceStub = Service.extend({
+      currentPage: {
+        title: 'Fresh Fish'
+      },
+      currentVersion: 'v3.2.0'
+    });
+    this.owner.register('service:page', pageServiceStub);
+
+    await render(hbs`{{guides-article model=model version=version currentVersion=currentVersion}}`);
+
+    let pageTitle = this.element.querySelector('h1');
```

Comment:
This is a probably a question for someone in the learning team, but since `qunit-dom` is not installed by default with ember-cli, do we want to use it in the examples. 

Reasons to do so:
- Assertions are way more readable, and often, less verbose.
- Arguably can be considered a good practice.

Reasons not to:
- People who like mocha might not like it or find it useful.
- `querySelector`, `querySelectorAll` and the usual qunit assertions (`ok`, `equal`), albeit more low lever, are also arguably easier to understand.